components/Tag.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import { StyleSheet } from 'react-native';
+import { Text } from 'react-native';
+
+export default class Tag extends Component<Props> {
+	constructor() {
+		super();
+	}
+
+	render() {
+		return (
+			<Text style={styles.tag}>#{this.props.tagText}</Text>
```

Comment:
Would defer to design but I think # is unneeded

Reply:
![image](https://user-images.githubusercontent.com/8290512/62081573-ed36cd00-b220-11e9-9071-e3814abcdec0.png)

designs show '#' should I keep or remove?